lib/src/html_generator.dart

```diff
@@ -221,61 +222,61 @@ class _HtmlGeneratorInstance implements HtmlOptions {
 
     generatePackage();
 
-    package.libraries.forEach((Library lib) {
+    for (var lib in package.libraries) {
```

Comment:
By using for loops, we avoid creating extra closures, which add to call stacks.

There's also nothing to step into when debugging – just a good practice to encourage and model for our users.